src/main/datatypes/MQSocialType.cpp

```diff
@@ -0,0 +1,120 @@
+/*
+ * MacroQuest: The extension platform for EverQuest
+ * Copyright (C) 2002-2023 MacroQuest Authors
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License, version 2, as published by
+ * the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ */
+
+#include "pch.h"
+
+#include "MQ2DataTypes.h"
+
+namespace mq::datatypes {
+
+//============================================================================
+
+#define MAX_SOCIAL 120
+#define MAX_CMD_LINES 5
```

Comment:
Oh, i missed these on my first review. we have SOCIAL_NUM_LINES and NUM_SOCIAL_PAGES, SOCIALS_PER_PAGE

Reply:
fixed.